source/configure/site-configuration-settings.rst

```diff
@@ -1091,6 +1100,75 @@ Message priority
+| notifications to.                                             | - ``config.json`` setting: ``.ServiceSettings.PersistentNotificationMaxRecipients: 5``  |
+|                                                               | - Environment variable: ``MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONSMAXRECIPIENTS``     |
+| Numerical input. Default is **5**.                            |                                                                                         |
++---------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+
+.. config:setting:: posts-frequencyofpersistentnotifications
+  :displayname: Frequency of persistent notifications (Posts)
+  :systemconsole: Site Configuration > Posts
+  :configjson: .ServiceSettings.PersistentNotificationIntervalMinutes
+  :environment: MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONINTERVALMINUTES
+  :description: The number of minutes between repeated notifications for urgent messages sent with persistent notifications. Default is **5**.
+
+Frequency of persistent notifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++---------------------------------------------------------------+-------------------------------------------------------------------------------------------+
+| The number of minutes between repeated notifications for      | - System Config path: **Site Configuration > Posts**                                      |
+| urgent messages sent with persistent notifications.           | - ``config.json`` setting: ``.ServiceSettings.PersistentNotificationIntervalMinutes: 5``  |
+|                                                               | - Environment variable: ``MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONINTERVALMINUTES``      |
+| Numerical input. Default is **5**.                            |                                                                                           |
++---------------------------------------------------------------+-------------------------------------------------------------------------------------------+
+
+.. config:setting:: posts-totalnumberofpersistentnotificationsperpost
+  :displayname: Total number of persistent notifications per post (Posts)
+  :systemconsole: Site Configuration > Posts
+  :configjson: .ServiceSettings.PersistentNotificationMaxCount
+  :environment: MM_SERVICESETTINGS_PERSISTENTNOTIFICATIONMAXCOUNT
+  :description: The maximum number of times users may receive persistent notifications. Default is **6**.
+
+Total number of persistent notifications per post
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++-------------------------------------------------------------+------------------------------------------------------------------------------------+
+| The maximum number of times users may receive persistent    | - System Config path: **Site Configuration > Posts**                               |
```

Comment:
@vish9812 is there a maximum allowable value for this? If not, should we add one to avoid abuse or other potential issues with infinite notifications

Reply:
Don't think there is any `max` value.
Although, it would be the `admin` who can abuse this value. So not sure, if we need to restrict `admins` too. :shrug:

Reply:
^ @esethna

Reply:
That's true, the admin has the control. No action needed here